api/v1beta1/ibmvpc_conversion.go

```diff
@@ -196,7 +196,7 @@ func Convert_v1beta2_IBMVPCMachineSpec_To_v1beta1_IBMVPCMachineSpec(in *infrav1b
 	return autoConvert_v1beta2_IBMVPCMachineSpec_To_v1beta1_IBMVPCMachineSpec(in, out, s)
 }
 
-func Convert_Slice_Pointer_string_To_Slice_Pointer_v1beta2_IBMVPCResourceReference(in *[]*string, out *[]*infrav1beta2.IBMVPCResourceReference, s apiconversion.Scope) error {
+func Convert_Slice_Pointer_string_To_Slice_Pointer_v1beta2_IBMVPCResourceReference(in *[]*string, out *[]*infrav1beta2.IBMVPCResourceReference, _ apiconversion.Scope) error {
```

Comment:
This is mostly autogenerated code, wondering if there is any action been taken to fix the code in the generator itself?

Reply:
and also wherever we are removing `old` var.

Reply:
I don't see any changes in the generator itself, the older version of lint is not detecting this discrepancies but the new one does. Renaming them as the actual function implementation contain those var even though we are not using them as of now wherever they are being referred.

Reply:
Let's create an issue in the generator repository so that at least they should know what's happening!

Reply:
Created a issue in generator (will update), the issues are being picked by newer lint versions only. I think this may have something to do with fixes in latest lint version itself. Meanwhile I guess we are good to merge this one for now.